docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,175 @@
+# Cluster to Advanced Cluster Migration Guide
+
+**Objective**: Guide users to replace the `mongodbatlas_cluster` resource with the `mongodbatlas_advanced_cluster` resource.
+
+-> **NOTE:** This guide focus on the resource migration as the data source migration is only requiring a resource_type change from `data.mongodbatlas_cluster` to `data.mongodbatlas_advanced_cluster`.  However, pay attention to the [output changes.](#output-changes)
+
+## Main Changes Between `mongodbatlas_cluster` and `mongodbatlas_advanced_cluster`
+
+Key differences in the configuration:
+
+1. Replication Spec Configuration: Supports different node types (electable, analytics, read_only) where hardware configuration can differ between node types.
```

Comment:
It might make sense to list these in the same order that they appear in the TF file itself. Also, in the code example, it might be nice to add commented out block headings like you have for "Provider Settings" just to make reference simple.

Reply:
Changed in 288f7f76a58ef30cf833e0adbb1e9543b719d138
WDYT?